components/match_ticker/commons/match_ticker_display_components_new.lua

```diff
@@ -0,0 +1,366 @@
+	end
+
+	local countdownArgs = {
+		date = dateString,
+		finished = match.finished
+	}
+
+	local countdownDisplay = mw.html.create('span')
+		:addClass('match-countdown')
+		:node(Countdown._create(countdownArgs))
+
+	return countdownDisplay
+end
+
+---@return Html?
+function Details:streams()
+	local match = self.match
+	local links = mw.html.create('div')
+		:addClass('match-streams')
+
+	if Table.isNotEmpty(match.stream) then
+		local streams = {}
+
+		-- Standardize the stream data to always use the platform as key (because of the new format ex: twitch_en_2)
+		for rawHost, stream in pairs(match.stream) do
+			local streamParts = mw.text.split(rawHost, '_', true)
+			if #streamParts == 3 then
+				local key = StreamLinks.StreamKey(rawHost)
+				streams[key.platform] = stream
+			else
+				streams[rawHost] = stream
+			end
+		end
+
+		local streamLinks = ''
+
+		-- Iterate over the streams and create the different links
+		for platformName, targetStream in pairs(streams) do
+			local streamLink = mw.ext.StreamPage.resolve_stream(platformName, targetStream)
+
+			if streamLink then
+				-- Default values
+				local url = 'Special:Stream/' .. platformName .. '/' .. streamLink
+				local icon = '<i class="lp-icon lp-icon-21 lp-' .. platformName .. '"></i>'
+
+				-- TL.net specific
+				if platformName == 'stream' then
+					url = 'https://tl.net/video/streams/' .. streamLink
+				end
+
+				streamLinks = streamLinks .. '[[' .. url .. '|' .. icon .. ']]'
+			end
+		end
+
+		links:wikitext(streamLinks)
+	end
+
+	return links
+end
+
+---@return Html?
+function Details:tournament()
+	if self.hideTournament then
+		return
+	end
+
+	local match = self.match
+
+	local icon = LeagueIcon.display{
+		icon = Logic.emptyOr(match.icon, TOURNAMENT_DEFAULT_ICON),
+		iconDark = match.icondark,
+		link = match.pagename,
+		name = match.tournament,
+		options = {noTemplate = true},
+	}
+
+	local displayName = Logic.emptyOr(
+		match.tickername,
+		match.tournament,
+		match.parent:gsub('_', ' ')
+	)
+
+	return mw.html.create('div')
+		:addClass('match-tournament')
+		:node(mw.html.create('div')
+			:addClass('tournament-icon')
+			:node(mw.html.create('div')
+				:wikitext(icon)
+			)
+		)
+		:node(mw.html.create('div')
+			:addClass('tournament-text')
+			:wikitext('[[' .. match.pagename .. '|' .. displayName .. ']]')
+		)
+
+end
+
+---Display class for matches shown within a match ticker
+---@class NewMatchTickerMatch
+---@operator call({config: MatchTickerConfig, match: table}): NewMatchTickerMatch
+---@field root Html
+---@field config MatchTickerConfig
+---@field match table
+local Match = Class.new(
+	function(self, args)
+		self.root = mw.html.create('div'):addClass('match')
+		self.config = args.config
+		self.match = args.match
+	end
+)
+
+---@return Html
+function Match:create()
+	self.root:node(self:standardMatchRow())
+	self.root:node(self:detailsRow())
+
+	return self.root
+end
```

Comment:
just as fyi: we are dropping BR support here, which atm is irrelevant since we don't have any BR support in match2 yet (except some edge cases on sc2 which i would want to exclude on match ticker anyways :P)

Reply:
Apex has match2 support

Reply:
Closing this, since we only focus on Dota2 right now